### ISSUE TITLE
Documentation did not build on docs.rs 

### ISSUE THREAD

**Issue by yangby-cryptape** [rust-lang/docs.rs#276]
- On my local machine, `cargo doc` works perfectly. 
  But the documents did not appeared on docs.rs.

  [`numext-constructor`] is proc-macro library for constructors.
  And I use those constructors to construct [`numext-fixed-uint`] and [`numext-fixed-hash`] (depend on [`numext-fixed-uint`]).

  First, I published [`numext-constructor`]; then, [`numext-fixed-uint`]; the last is [`numext-fixed-hash`].

  **BUT**, the document for [`numext-fixed-hash`] wasn't built, the documents for other two crates are ok.
  I do `cargo yank` then `cargo publish` again, the document for [`numext-fixed-hash`] still wasn't built.

- And, is there a way to rebuild missing version of crates ([`numext-fixed-hash`] v0.1.1 and v0.1.2)?

[`numext-constructor`]: https://crates.io/crates/numext-constructor
[`numext-fixed-uint`]: https://crates.io/crates/numext-fixed-uint
[`numext-fixed-hash`]: https://crates.io/crates/numext-fixed-hash

**Comment by onur** [comment]
Hi @yangby-cryptape, thanks for reporting. I am seeing this error in the build log:

```
2018/12/18 04:48:48 [INFO] cratesfyi::docbuilder::chroot_builder: Building package numext-fixed-hash-0.1.2
2018/12/18 04:48:48 [DEBUG] cratesfyi::utils::build_doc: Getting package with cargo
    Updating crates.io index
 Downloading crates ...
  Downloaded numext-fixed-hash v0.1.2
2018/12/18 04:48:48 [ERROR] cratesfyi::docbuilder::queue: Failed to build package numext-fixed-hash-0.1.2 from queue: failed to parse manifest at `.../.cargo/registry/src/github.com-1ecc6299db9ec823/numext-fixed-hash-0.1.2/Cargo.toml`
```

This is the generated file when you upload your crate to crates.io:

```
# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
#
# When uploading crates to the registry Cargo will automatically
# "normalize" Cargo.toml files for maximal compatibility
# with all versions of Cargo and also rewrite `path` dependencies
# to registry (e.g. crates.io) dependencies
#
# If you believe there's an error in this file please file an
# issue against the rust-lang/cargo repository. If you're
# editing this file be aware that the upstream Cargo.toml
# will likely look very different (and much more reasonable)

[package]
edition = "2018"
name = "numext-fixed-hash"
version = "0.1.2"
authors = ["Boyu Yang <yangby@cryptape.com>", "zhangsoledad <787953403@qq.com>"]
description = "Fixed-size hash types."
homepage = "https://yangby-cryptape.github.io/rust-numext/"
keywords = ["fixed", "hash", "bignum"]
categories = ["algorithms", "data-structures"]
license = "Apache-2.0 OR MIT"
repository = "https://github.com/yangby-cryptape/rust-numext"
[dependencies.failure]
version = "0.1"

[dependencies.faster-hex]
version = "0.1"
optional = true

[dependencies.heapsize]
version = "0.4"
optional = true

[dependencies.nfuint]
version = "~0.1.2"
package = "numext-fixed-uint"

[dependencies.numext-constructor]
version = "~0.1.2"

[dependencies.rand]
version = "0.5"
optional = true

[dependencies.serde]
version = "1.0"
optional = true

[features]
default = []
support_all = ["support_rand", "support_heapsize", "support_serde"]
support_heapsize = ["heapsize", "nfuint/support_heapsize"]
support_rand = ["rand", "nfuint/support_rand"]
support_serde = ["serde", "faster-hex", "nfuint/support_serde"]
[badges.travis-ci]
repository = "yangby-cryptape/rust-numext"
```

I am not sure why docs.rs is failing to parse this file. Maybe @QuietMisdreavus can figure out.

**Comment by QuietMisdreavus** [comment]
I'm not totally sure what happened either. That kind of "failed to parse manifest" error can come if you're using a feature in cargo that's newer than the cargo library we're using - i remember hitting that when people were using the edition feature right after it stabilized on nightly.

However, i just tried building it myself to get a build log, and it worked for me `>_>` https://docs.rs/numext-fixed-hash/0.1.2/numext_fixed_hash/

I think we just updated our cargo library via https://github.com/rust-lang/docs.rs/pull/271 so the problem may have fixed itself.

**Comment by yangby-cryptape** [comment]
Thanks!

I will check if it works when I publish again.

In the near future, I have no plan to publish new versions of these crates.
So, I close this issue, for now.

**Comment by yangby-cryptape** [comment]
Hi, the documents for both [`numext-fixed-hash v0.1.1`] and [`numext-fixed-hash v0.1.2`] are missing, again.

[`numext-fixed-hash v0.1.1`]: https://docs.rs/numext-fixed-hash/0.1.1/numext_fixed_hash
[`numext-fixed-hash v0.1.2`]: https://docs.rs/numext-fixed-hash/0.1.2/numext_fixed_hash

**Comment by onur** [comment]
@yangby-cryptape recent server migration rolled back your crates (and maybe few others recently built manually in the old server). I rebuilt your crates and it should work fine now.
